storage/pebble/operation/common_test.go

```diff
@@ -1,23 +1,45 @@
-// (c) 2019 Dapper Labs - ALL RIGHTS RESERVED
-
 package operation
 
 import (
 	"bytes"
+	"encoding/hex"
 	"fmt"
-	"reflect"
 	"testing"
 
-	"github.com/dgraph-io/badger/v2"
+	"github.com/cockroachdb/pebble"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"github.com/vmihailenco/msgpack/v4"
+	"github.com/vmihailenco/msgpack"
 
-	"github.com/onflow/flow-go/model/flow"
 	"github.com/onflow/flow-go/storage"
 	"github.com/onflow/flow-go/utils/unittest"
 )
 
+var upsert = insert
+var update = insert
```

Comment:
We are not able to support `update` with pebble. 

The update operation in badger will first check if the key exists, and only update if the key does not exist. 

However, if we implement the similar logic in pebble, it won't be concurrent-safe anyway, because a read can be dirty without transaction. 

For instance, if one operation is to update a key with value 10, and another operation is to update the same key with value 20. And if these operations happen concurrently, then both of them might see the key does not exist, and continue to set the value, and in the end, both operation would succeed, and the value will end up being either 10 or 20 depending on which lose the race.

That's why I decided to not implement `update` and just use `insert` everywhere, the insert will simply set the value regardless the key exists or not.

Reply:
I agree with removing `upsert` and `update` at the low level. 👍 

Before merging, I think we should remove these aliases and remove any usages of the functions (I only found one, in `TestUpsertEntry`).